resources/istio/templates/istio-operator.yaml

```diff
@@ -34,6 +34,7 @@ spec:
         serviceAnnotations:
           prometheus.io/scrape: "true"
           prometheus.io/port: "15014"
+          service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout: "4000"
```

Comment:
Why do we add it to the pilot configuration? It must be added to the annotations of [istio-ingressgateway](https://github.com/kyma-project/kyma/blob/26c81b47eb353fb7fbcaa0709d97ee052a760d0c/resources/istio/templates/istio-operator.yaml#L61).

Reply:
You have good eyes Tim, thank you for spotting my failure 🫣